packages/perseus/src/types.js

```diff
@@ -9,7 +9,7 @@ import * as React from "react";
 import type {SerializedHighlightSet} from "./components/highlighting/types.js";
 import type {ILogger} from "./logging/log.js";
 import type {Item} from "./multi-items/item-types.js";
-import type {PerseusWidget} from "./perseus-types.js";
+import type {PerseusWidget, PerseusRadioChoice} from "./perseus-types.js";
```

Comment:
What's the difference between `types.js` and `perseus-types.js`? I imagine one is externally facing and one is internally facing?

Reply:
`perseus-types.js` is the file that contains mostly types that model the JSON that gets published (ie. Perseus data). `types.js` is the file containing types that Perseus uses (internally, for the most part).

Reply:
We should probably add a comment at the top of `perseus-types.js` and/or rename the file to `perseus-json-types.js` or something like that to make it more obvious that these types describe the data that's in data store.